ludwig/features/audio_feature.py

```diff
@@ -359,7 +356,7 @@ def add_feature_data(
 
         if feature_config[PREPROCESSING]["in_memory"]:
             audio_features = AudioFeatureMixin._process_in_memory(
-                input_df[feature_config[NAME]],
+                input_df[column],
```

Comment:
Why has this changed? What is in `input_df[feature_config[NAME]]`?

Reply:
I just changed that for consistency since it was getting defined above already and used in a few other places. `COLUMN` and `NAME` are really the same thing in most cases, but in the few edge cases I believe it should be using column instead of name to index the df.